glib/tests/clone.rs

```diff
@@ -451,29 +451,46 @@ fn test_clone_macro_typed_args() {
 
```

Comment:
Maybe the tests just above can also return something from their closures that can be tested?

Reply:
Another PR incoming then!